### ISSUE TITLE
When restarting a workflow, it doesn't go to processing state and it returns empty workflow object

### ISSUE THREAD

**Issue by drjova** [cern-sis/issues-inspire#647]

**Comment by drjova** [comment]
![Screenshot 2024-12-19 at 15 59 50](https://github.com/user-attachments/assets/4e0a5fd5-6885-48f5-907f-00499bb3c9b9)

**Comment by DonHaul** [comment]
where is this restart coming from? if its  from https://workflows.inspire.dev.siscern.org/ it seems that its going back to processing

**Comment by DonHaul** [comment]
when restarting the workflow via the `restart workflow` button in https://inspirebeta.net/backoffice/6972fef5-4a4a-4191-a4b9-4c728f06a53e the workflow gets restarted without the "workflow" field

**Comment by drjova** [comment]
Yes, it’s from the back office, but the interface doesn’t have the correct state. And if you refresh, you are still getting an error instead of processing.

**Comment by drjova** [comment]
TECH NOTES:
* [Here](https://github.com/inspirehep/inspirehep/blob/master/backoffice/backoffice/authors/api/views.py#L221) we should change the status like in [resolve](https://github.com/inspirehep/inspirehep/blob/master/backoffice/backoffice/authors/api/views.py#L195-L200) and respond with the worklow data
